base/src/main/java/org/aya/core/def/FieldDef.java

```diff
@@ -33,4 +34,13 @@ public FieldDef(
   public @NotNull DefVar<FieldDef, TeleDecl.StructField> ref() {
     return ref;
   }
+
+  private @NotNull SortTerm structResult() {
+    return structRef.concrete != null ? structRef.concrete.ulift : structRef.core.result;
+  }
+
+
+  public boolean inProp() {
+    return structResult().isProp();
+  }
```

Comment:
Is it possible to turn this into SubLevelDef

Reply:
SubLevel doesn’t have structRef

Reply:
I believe there will be a way to share some code

Reply:
It might not be trivial